conf/nginx/templates/default.conf.template

```diff
@@ -40,6 +40,11 @@ server {
     ssl_certificate     /etc/ssl/certs/portal.cer;
     ssl_certificate_key /etc/ssl/private/portal.key;
 
+    add_header Permissions-Policy "autoplay=(), camera=(), encrypted-media=(), fullscreen=(), geolocation=(), gyroscope=(), magnetometer=(), microphone=(), midi=(), payment=()" always;
```

Comment:
The only concern I can imagine is `geolocation` not allowing a user to get directions via embedded Google map. But even [TACC site (Contact page)](https://www.tacc.utexas.edu/about/contact/) (not deployed via Camino) does not offer that.

Reply:
After reading [w3c's permissionspolicy explainer](https://github.com/w3c/webappsec-permissions-policy/blob/main/permissions-policy-explainer.md#example-2---enable-geolocation-in-a-cross-origin-frame), I think even [that use case I imagined](https://github.com/TACC/Camino/pull/31/files#r1259955796) is an issue we can (and should) manage as needed via ` allow="geolocation"` attribute in `<iframe>`.

Reply:
@wesleyboar  - Thanks for testing this. 
Regarding geolocation, you are right that tacc site redirects to google maps for getting directions and does not use geolocation api.
I'm leaning towards more restrictive security settings, and relaxing them based on feature requirements. If there is no current feature usage, we can leave it as is.

Reply:
👍 I lean toward that also.*

<sub>* Because NSO wants me to lean toward that with CMS user permissions.</sub>